### PR TITLE
Update scale-tidb-using-tiup.md

### PR DESCRIPTION
**Pull request by easonn7**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by WalterWj** [comment]
@easonn7 Any update ?

File: scale-tidb-using-tiup.md [hunk 1]
```diff
@@ -327,7 +349,29 @@ tiup cluster scale-in <cluster-name> --node 10.0.1.5:20160
...
+
+（1）更新集群配置：
+
+  {{< copyable "shell-regular" >}}
+
+  ```shell
+  tiup cluster reload <cluster-name> --skip-restart
+  ```
+
+（2）更新 Prometheus 配置并重启：
```
Comment (kaaaaaaang) [inline scale-tidb-using-tiup.md:366]: 从 tiup cluster v1.15.0 起不再需要第二步了，删除这一小段吧

File: scale-tidb-using-tiup.md [hunk 2]
```diff
@@ -128,7 +128,29 @@ pd_servers:
...
```
[the remark this answers is not in the document]
Comment (qiancai) [reply scale-tidb-using-tiup.md:145]: 要不修改为一个 Note，如果用户的 tiup 版本低于 v1.15.0，就需要更新 Prometheus 配置并重启

**Comment by WalterWj** [comment]
Can this pr be merged as soon as possible?

File: scale-tidb-using-tiup.md [hunk 3]
```diff
@@ -128,7 +128,29 @@ pd_servers:
...
+>
+> 该操作仅在 PD 扩容时需要被执行。
+
+（1）更新集群配置：
+
+  {{< copyable "shell-regular" >}}
+
+  ```shell
+  tiup cluster reload <cluster-name> --skip-restart
+  ```
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:143]:
````suggestion
1. 更新集群配置：

    ```shell
    tiup cluster reload <cluster-name> --skip-restart
    ```
````

File: scale-tidb-using-tiup.md [hunk 4]
```diff
@@ -128,7 +128,29 @@ pd_servers:
...
+
+（2）更新 Prometheus 配置并重启：
+
+  {{< copyable "shell-regular" >}}
+
+  ```shell
+  tiup cluster reload <cluster-name> -R prometheus
+  ```
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:151]:
````suggestion
2. 更新 Prometheus 配置并重启：

    > **注意：**
    >
    > 如果你使用的 TiUP v1.15.0 及之后版本，请跳过此步骤；如果你使用的 TiUP 版本早于 v1.15.0，则需要执行以下命令来更新 Prometheus 配置并重启。

    ```shell
    tiup cluster reload <cluster-name> -R prometheus
    ```
````

File: scale-tidb-using-tiup.md [hunk 5]
```diff
@@ -128,7 +128,29 @@ pd_servers:
...
+
+### 4. 检查集群状态
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:153]: 与其它小节的用词保持一致
```suggestion
### 4. 查看集群状态
```

File: scale-tidb-using-tiup.md [hunk 6]
```diff
@@ -327,7 +349,29 @@ tiup cluster scale-in <cluster-name> --node 10.0.1.5:20160
...
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:374]:
```suggestion
### 4. 查看集群状态
```

File: scale-tidb-using-tiup.md [hunk 7]
```diff
@@ -327,7 +349,29 @@ tiup cluster scale-in <cluster-name> --node 10.0.1.5:20160
 
 预期输出 Scaled cluster `<cluster-name>` in successfully 信息，表示缩容操作成功。
 
-### 3. 检查集群状态
+### 3. 刷新集群配置
+
+> **注意：**
+>
+> 该操作仅在 PD 缩容时需要被执行。
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:356]:
```suggestion
> 该操作仅需在缩容 PD 节点时执行，缩容 TiDB 或 TiKV 节点时无需执行。
```

File: scale-tidb-using-tiup.md [hunk 8]
```diff
@@ -128,7 +128,29 @@ pd_servers:
 
 预期日志结尾输出 ```Scaled cluster `<cluster-name>` out successfully``` 信息，表示扩容操作成功。
 
-### 3. 检查集群状态
+### 3. 刷新集群配置
+
+> **注意：**
+>
+> 该操作仅在 PD 扩容时需要被执行。
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:135]:
```suggestion
> 该操作仅需在扩容 PD 节点时执行，扩容 TiDB 或 TiKV 节点时无需执行。
```

File: scale-tidb-using-tiup.md [hunk 9]
```diff
@@ -327,7 +349,29 @@ tiup cluster scale-in <cluster-name> --node 10.0.1.5:20160
...
+>
+> 该操作仅在 PD 缩容时需要被执行。
+
+（1）更新集群配置：
+
+  {{< copyable "shell-regular" >}}
+
+  ```shell
+  tiup cluster reload <cluster-name> --skip-restart
+  ```
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:364]:
````suggestion
1. 更新集群配置：

    ```shell
    tiup cluster reload <cluster-name> --skip-restart
    ```
````

File: scale-tidb-using-tiup.md [hunk 10]
```diff
@@ -327,7 +349,29 @@ tiup cluster scale-in <cluster-name> --node 10.0.1.5:20160
...
+
+（2）更新 Prometheus 配置并重启：
+
+  {{< copyable "shell-regular" >}}
+
+  ```shell
+  tiup cluster reload <cluster-name> -R prometheus
+  ```
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:372]:
````suggestion
2. 更新 Prometheus 配置并重启：

    > **注意：**
    >
    > 如果你使用的 TiUP v1.15.0 及之后版本，请跳过此步骤；如果你使用的 TiUP 版本早于 v1.15.0，则需要执行以下命令来更新 Prometheus 配置并重启。

    ```shell
    tiup cluster reload <cluster-name> -R prometheus
    ```
````

File: scale-tidb-using-tiup.md [hunk 11]
```diff
@@ -327,7 +349,29 @@ tiup cluster scale-in <cluster-name> --node 10.0.1.5:20160
...
+
+    ```shell
+    tiup cluster reload <cluster-name> --skip-restart
+    ```
+
+2. 更新 Prometheus 配置并重启：
+
+    > **注意：**
+    >
+    > 如果你使用的 TiUP v1.15.0 及之后版本，请跳过此步骤；如果你使用的 TiUP 版本早于 v1.15.0，则需要执行以下命令来更新 Prometheus 配置并重启。
```
Comment (qiancai) [inline scale-tidb-using-tiup.md:368]:
```suggestion
    > 如果你使用的是 TiUP v1.15.0 及之后版本，请跳过此步骤；如果你使用的 TiUP 版本早于 v1.15.0，则需要执行以下命令来更新 Prometheus 配置并重启。
```

**Review by kaaaaaaang — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by qiancai — APPROVED** [review]

**Pull request merged**